README.md

```diff
@@ -1 +1,195 @@
-# projectAwesome
\ No newline at end of file
+# projectAwesome
+
+## Description
+
+The tasks is to design and build an app that combines a robust back end--with servers, databases, advanced APIs, and user authentication—-to an intuitive front end. we were to use the MVC paradigm, create our own server-side API,  have user authentication, and connect to a database.
+
+## User Story
+
+As a Racing Enthusiast, I want to be able to sign up to participate in races with my vehicle, add my vehicles to my garage with their specs also be able to register for a race and see the races that I have signed my cars up for.  
+
+As a Track Owner, I want to be able to post track events. Once its posted, I would like to be able to manage my posting by editing and or deleting posted events.
+
+
+## Acceptance Criteria
+(Need to update)
```

Comment:
WHEN I go to the page but am not authenticated, 
THEN I am sent to the login page.
WHEN I log in
THEN I am directed to the homepage where I can view available races and sign my cars up for a race.
WHEN I go to my garage
THEN I can see the cars I have and the races they are signed up for
WHEN I go to my garage
THEN I can add cars to my garage
WHEN I add a car
THEN the car and specs are loaded to the web page
WHEN I create a race
THEN I can see the race details in my garage and it is available on the homepage for others to sign up for
WHEN I update a car or a race
THEN the updated details are populated to my garage and homepage.
WHEN I delete a car or race
THEN the car or race is removed from the garage an homepage